core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -288,4 +304,44 @@ class SpendReportingService(
     }
   }
 
+  def getSpendForBillingProject(
+    project: RawlsBillingProjectName,
+    start: DateTime,
+    end: DateTime,
+    aggregations: Set[SpendReportingAggregationKeyWithSub]
+  ): Future[SpendReportingResults] =
+    for {
+      billingProject <- billingRepository.getBillingProject(project)
+
+      report <- getReportData(billingProject.get, project, start, end, aggregations)
+
+      result = report
+    } yield result
```

Comment:
We can return `report` directly

```suggestion
    } yield report
```